right/src/led_display.c

```diff
@@ -0,0 +1,11 @@
+#include "led_display.h"
+#include "layer.h"
+
+#define LAYER_LED_FIRST    FRAME_REGISTER_PWM_FIRST + 13
+#define LAYER_LED_DISTANCE 16
+
+void LedDisplay_SetLayerLed(uint8_t layerId, uint8_t brightness) {
```

Comment:
Remove the `brightness` parameter and use `LedDisplayBrightness` in the function body according to the above.